api/indexer/client.go

```diff
@@ -32,15 +32,15 @@ type Client struct {
 // Use a separate type that only decodes the block bytes because we cannot decode block JSON
 // due to Actions/Auth interfaces included in the block's transactions.
 type GetBlockClientResponse struct {
-	BlockBytes codec.Bytes `json:"blockBytes"`
+	BlockBytes codec.Bytes `json:"block"`
 }
 
 func (c *Client) GetBlock(ctx context.Context, blkID ids.ID, parser chain.Parser) (*chain.ExecutedBlock, error) {
 	resp := GetBlockClientResponse{}
 	err := c.requester.SendRequest(
 		ctx,
 		"getBlock",
-		&GetBlockRequest{BlockID: blkID},
+		&GetBlockRequest{BlockID: blkID, Encoding: Hex},
```

Comment:
I did not provide the Encoding through variable, I think this client will continue to always use `Hex`.